jetty-plus/src/main/java/module-info.java

```diff
@@ -29,14 +29,19 @@
 
     requires java.naming;
     requires java.transaction;
-    requires org.eclipse.jetty.util;
     requires org.eclipse.jetty.jndi;
     requires org.eclipse.jetty.security;
+    requires org.eclipse.jetty.util;
     requires org.eclipse.jetty.webapp;
     requires org.eclipse.jetty.xml;
+
+    // Only required if using DataSourceLoginService.
     requires static java.sql;
+    // Only required if using ContainerInitializer.
     requires static jetty.servlet.api;
+    // Only required if using DataSourceLoginService.
     requires static org.eclipse.jetty.server;
+    // Only required if using RunAs.
```

Comment:
I believe there is no valid use case for using jetty-plus without the servlet spec.
jndi and initial context were mentioned, but those can be used without jetty-plus as well, just relying on the default implementation in the JVM.
the jndi and initial context provided by jetty-plus are for a servlet specific behaviors and requirements.